src/backend/joanie/core/flows/order.py

```diff
@@ -80,6 +83,25 @@ def to_sign(self):
         Transition order to to_sign state.
         """
 
+    def _can_be_state_signing(self):
+        """
+        An order state can be set to signing if the order has an unsigned contract.
```

Comment:
IMO, maybe we should add a more descriptive docstring since we have submitted the contract to sign but we are waiting for the return from the signature provider through the webhook to confirm that the student has signed the document. The docstring feels a bit similar with the method `_can_be_state_to_sign`